ddtrace/contrib/pytest/_utils.py

```diff
@@ -104,3 +123,24 @@ def _is_pytest_8_or_later() -> bool:
 
 def _pytest_version_supports_itr() -> bool:
     return _get_pytest_version_tuple() >= ITR_MIN_SUPPORTED_VERSION
+
+
+def _pytest_marked_to_skip(item: pytest.Item) -> bool:
+    """Checks whether Pytest will skip an item"""
+    if item.get_closest_marker("skip") is not None:
+        return True
+
+    return any([True for marker in item.iter_markers(name="skipif") if marker.args[0] is True])
```

Comment:
@romainkomorndatadog Can we rewrite this as:
```suggestion
    return any(marker.args[0] is True for marker in item.iter_markers(name="skipif"))
```
?

Reply:
I think we can actually even rewrite it as
```suggestion
    return any(marker.args[0] for marker in item.iter_markers(name="skipif"))
```
because `skipif` will skip if the first argument is truthy:
```
@pytest.mark.skipif(lambda: 'true', reason="true string")
```
results in
```
tests/test_views.py::test_remove_method_from_parent SKIPPED (true string)
```

I'll make the change in my soon-to-come follow-up.